jupyterhub/tests/test_singleuser.py

```diff
@@ -63,6 +65,8 @@ async def test_singleuser_auth(
         await user.spawn(server_name)
         await app.proxy.add_user(user, server_name)
     spawner = user.spawners[server_name]
+    if IS_JUPYVERSE:
+        spawner.default_url = "/lab"
```

Comment:
What happens if this is missing? Is it because the default url for jupyverse is not `/tree`?

Reply:
Yes, just like JupyterLab has no `/tree` endpoint.